src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work.");

```

Comment:
The message should indicate the specific type of `Machine` that is working, including the class name. Consider changing the message to something like 'Truck is working.' to make it more informative.